src/settings/mod.rs

```diff
@@ -1,5 +1,15 @@
 //! Configure your application.
 
+#[cfg(target_os = "windows")]
+#[path = "windows.rs"]
+pub mod platform;
+
+#[cfg(not(target_os = "windows"))]
+#[path = "not_windows.rs"]
+pub mod platform;
+
+pub use platform::PlatformSpecific;
```

Comment:
I would only expose `PlatformSpecific` settings in `iced_winit` for the time being.

Reply:
Sure.
I made iced's platform-specific settings use iced_winit's `PlatformSettings`.